types/figma/index.d.ts

```diff
@@ -713,3 +808,6 @@ interface Image {
     readonly hash: string;
     getBytesAsync(): Promise<Uint8Array>;
 }
+} // declare global
+
+export {};
```

Comment:
Just curious, why the change to a global-augmenting module from the global declarations?

Reply:
The global `figma` variable was no longer being automatically picked up by `tsc` and needed an import statement, this happened to fix it.

Reply:
I would think that _after_ this change, it would require an `import 'figma'` to get the globals included. If you’re trying to get the globals to show up without an import declaration, can you send me a repro of it not being picked up so I can dig into that?